domain/test/db.async.test.js

```diff
@@ -128,24 +128,24 @@ const coll = 'tests';
             assert(returned2[0].type === 'axe');
         });
 
-        it('should be fully to update item', async function () {
+        it('should replace item', async function () {
             let obj = { qwer: 11, type: 'axe' };
             let id = await database.add(db, coll, obj);
             obj.type = 'sword';
             delete obj.qwer;
-            let n = await database.updateItemFully(db, coll, id, obj);
-            assert(n === 1);
+            let repRes = await database.replaceById(db, coll, id, obj);
+            assert(repRes === true);
             let res = await database.getById(db, coll, id);
             assert(res.type === 'sword');
         });
 
-        it('should be error at update item', async function () {
+        it('should be error at replace item', async function () {
```

Comment:
I've already told you about it, haven't I? 
This is not descriptive. You need to specify condition of not-replacing. Moreover, it is just plain out wrong - there is no error, item just doesn't get replaced.
So the correct title is "should not replace non-existing item", as I already said earlier 👿 https://github.com/emotz/kladovka/pull/75#discussion_r125813445. And you should assert that the item is indeed still missing in db after trying to replace.

The reasoning is that you have two tests one after another - "should replace item", "should be error to replace item". They are controversial - both state the opposite things. And what if there is another condition to fail replacing? What would you name it? "should be error to replace item 2"? And what if there is 10 "failing" tests?